target/linux/ramips/dts/TL-WR802NV4.dts

```diff
@@ -0,0 +1,52 @@
+/dts-v1/;
```

Comment:
@majo-icutech Please can you add here [license specifier](https://github.com/openwrt/openwrt/blob/master/target/linux/ramips/dts/BDCOM-WAP2100-SK.dts#L1)? For some example just do `git grep SPDX target/linux/ramips/dts`